src/runtime/meter-toast-api.js

```diff
@@ -29,6 +29,16 @@ export const IFRAME_BOX_SHADOW =
   'rgba(60, 64, 67, 0.3) 0px -2px 5px, rgba(60, 64, 67, 0.15) 0px -5px 5px';
 export const MINIMIZED_IFRAME_SIZE = '420px';
 
+// If the user is able to close the dialog before loading is complete,
+// this timeout ensures we still pingback a metering entitlement after X ms.
+// This timeout should represent the longest time it could reasonably take
+// to load a SwG BOQ iframe.
+const AUTO_PINGBACK_TIMEOUT = 10000;
+
+function getScrollY(win) {
```

Comment:
Why did you move move this from `this.win_. pageYOffset`? Is this for testing?

Reply:
It didn't work on my computer until I did this.  I could not get it to pass tests